### PR TITLE
install the agent in the global-hub-system ns

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: pkg/constants/constants.go [hunk 1]
```diff
@@ -55,10 +55,9 @@ const (
 	HubClusterClaimName = "hub.open-cluster-management.io"
 
 	// the value of the HubClusterClaimName ClusterClaim
-	HubNotInstalled                   = "NotInstalled"
-	HubInstalledWithSelfManagement    = "InstalledWithSelfManagement"
-	HubInstalledWithoutSelfManagement = "InstalledWithoutSelfManagement"
-	HubInstalledByHoH                 = "InstalledByHoH"
+	HubNotInstalled    = "NotInstalled"
+	HubInstalledByUser = "InstalledByUser"
+	HubInstalledByHoH  = "InstalledByHoH"
```
Comment (clyang82) [inline pkg/constants/constants.go:60]: suggest to use `InstalledByGlobalHub`

File: agent/pkg/controllers/controllers_integration_test.go [hunk 2]
```diff
@@ -80,7 +80,7 @@ var _ = Describe("controller", Ordered, func() {
 			}, clusterClaim)
 		}, 1*time.Second, 100*time.Millisecond).ShouldNot(HaveOccurred())
 		Expect(clusterClaim.Spec.Value).Should(Equal(
-			constants.HubInstalledWithoutSelfManagement))
+			constants.HubInstalledByUser))
```
Comment (clyang82) [inline agent/pkg/controllers/controllers_integration_test.go:83]: that means we do not care about if the hub cluster is installed with self management or not. correct?

File: pkg/constants/constants.go (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/constants/constants.go]: done

File: agent/pkg/controllers/controllers_integration_test.go (hunk 2 above)
Comment (zhiweiyin318) [reply agent/pkg/controllers/controllers_integration_test.go:83]: yes. we do not need to detect the local-cluster.

**Pull request merged**